src/connector/src/sink/big_query.rs

```diff
@@ -743,7 +743,7 @@ fn build_protobuf_descriptor_pool(desc: &DescriptorProto) -> prost_reflect::Desc
     prost_reflect::DescriptorPool::from_file_descriptor_set(FileDescriptorSet {
         file: vec![file_descriptor],
     })
-    .unwrap()
+    .map_err(|err| SinkError::BigQuery(anyhow::anyhow!("Build descriptor pool error: {:?}", err)))
```

Comment:
Please follow the style like this:

https://github.com/risingwavelabs/risingwave/blob/f5ae55d29f416a4dd5ba01e81e727296da645c59/src/connector/src/sink/snowflake.rs#L246-L247